charms/jupyter-controller/requirements.in

```diff
@@ -1,2 +1,4 @@
+charmed-kubeflow-chisme
+lightkube
```

Comment:
are these two necessary?  maybe they came in as a copy/paste?  I don't think the charm needs these

Reply:
Will try it out. Was just porting the solution.

Reply:
Done.